client/wfnews-war/src/main/angular/src/app/utils/index.ts

```
@@ -806,7 +806,9 @@ export function formatDate(timestamp: string | number): string {
     };
 
     return date.toLocaleDateString('en-US', options);
-  } else return '';
```

should we throw an exception instead of a blank string?

---

Yes, updated now